fleet/lib/rook-ceph-conf/charts/konkong/templates/cephobjectstore-s3-butler.yaml

```diff
@@ -2,22 +2,22 @@
 apiVersion: ceph.rook.io/v1
 kind: CephObjectStore
 metadata:
-  name: latiss-raw
+  name: s3-butler
   namespace: rook-ceph
 spec:
   metadataPool:
     failureDomain: host
     replicated:
       size: 3
     quotas:
-      maxSize: 50Gi
+      maxSize: 2.1Ti
   dataPool:
     failureDomain: host
     erasureCoded:
       dataChunks: 2
       codingChunks: 1
     quotas:
-      maxSize: 1.01Ti
+      maxSize: 42.01Ti
```

Comment:
```suggestion
      maxSize: 42Ti
```

Reply:
That's great, thanks I will take those suggestions, I recall a while ago you were putting a little extra in some configurations quotes.